pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
in any case, I'd want tests through the proxy to a backend using no transport encoding, gzip, and chunked, with and without content-length, to make sure the backend gets a correct content length in all cases